components/nuclio/README.md

```diff
@@ -0,0 +1,44 @@
+# Nuclio (Serverless) Components
+
+[Nuclio](https://nuclio.io/) is a native and high performance serverless platform over Kubernetes
+which automate the process of build, deployment, monitoring, and auto-scaling of micro-services.
+Nuclio support variety of data and data-science related features (e.g. stream processing, 
+GPUs, volume/DB mounts, high concurrency, etc.)
+
+To install Nuclio over Kubernetes follow the [instruction in Github](https://github.com/nuclio/nuclio), 
+or this [interactive tutorial](https://www.katacoda.com/javajon/courses/kubernetes-serverless/nuclio).
+
```

Comment:
Are we supposed to install Nuclio on Kube before using these components?

Reply:
@animeshsingh it can be installed on the same cluster or a different one (defined by the dashboard address)

Reply:
i plan on adding a page under `kubeflow` with more detailed install/usage using `helm`